### DISCUSSION TITLE
How to use /tmp in NodeJS as temporary storage?

### DISCUSSION THREAD

**Discussion by Usama3627** [deta/cloud-docs#541] [Help]
As mentioned in the docs, 512MB of /tmp storage is available. I tried to use it but I get the following error:

EROFS: read-only file system, open 'tmp/img/1.png'

It should be allowed as far as I know. It was in the root directory of my project. Can anyone please tell me how to access it?

**Comment by lemonyte** [comment]
The directory is `/tmp` with a leading slash, because it isn't relative to your project's root dir.
